src/popup/imageDetailModule.js

```diff
@@ -81,6 +81,61 @@ export function getHtmlAttribution(image) {
   return `<p style="font-size: 0.9rem;font-style: italic;">${imgLink}${creator}${licenseLink}${licenseImgLink}</p>`;
 }
 
+/**
+ * @desc Helps in Formatting HTML Attributions
+ * @param {string} code - HTML Attributions.
+ */
+function formatCode(code, stripWhiteSpaces, stripEmptyLines) {
```

Comment:
Please name the function `formatHTMLAttribution`.

Reply:
Done, Will take care of `allow collaborators to pus`h from next time onwards